ocp4/profiles/cis.profile

```diff
@@ -0,0 +1,105 @@
+documentation_complete: true
```

Comment:
Let's not add this profile just yet. It's using OCP3 only checks and won't work in OCP4.